src/actions/hasErrored.test.js

```diff
@@ -0,0 +1,16 @@
+import * as actions from '../actions'
```

Comment:
The star import looks reallypowerful!

Reply:
Yeah that is a good tool to use from now on.